### PR TITLE
🆕🎨 Design updates V

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

General design updates:
- Swap container mobile optimizations
- Style fee warnings
- Appzi button styling for mobile
- Footer mobile/tablet optimizations
- Network selector mobile optimization

Screen Shot 2022-11-22 at 13 00 24Screen Shot 2022-11-22 at 13 00 17
Screen Shot 2022-11-22 at 13 00 09

### CODE REVIEW

File: src/custom/components/SwapWarnings/index.tsx [hunk 1]
```diff
@@ -58,26 +78,26 @@ const WarningContainer = styled(AuxInformationContainer).attrs((props) => ({
...
+    > svg:first-child {
+      stroke: var(--warningColor);
     }
   }
 `
 
 const ErrorStyledInfoIcon = styled(StyledInfoIcon)`
   color: ${({ theme }) => theme.infoText};
 `
-const HIGH_TIER_FEE = { level: '30', colour: '#FF7676' }
```
Comment (fairlighteth) [inline src/custom/components/SwapWarnings/index.tsx:70]: Moving the color selection to the styled component. Instead only feed the `level` number.

File: src/custom/components/SwapWarnings/index.tsx [hunk 2]
```diff
@@ -39,15 +39,35 @@ const WarningContainer = styled(AuxInformationContainer).attrs((props) => ({
   ...props,
   hideInput: true,
 }))<HighFeeContainerProps>`
-  background: ${({ theme, bgColour }) => bgColour || theme.info};
-  color: ${({ theme, textColour }) => textColour || theme.infoText};
-  padding: ${({ padding = '10px' }) => padding};
+  --warningColor: ${({ theme, level }) =>
```
Comment (fairlighteth) [inline src/custom/components/SwapWarnings/index.tsx:42]: Store color in a CSS VAR based on the `level` prop

**Comment by elena-zh** [comment]
Hey @fairlighteth , great!
Some my notes: 
1. Tablet view: the last line in the footer menu is hidden, need to scroll down
tablet - need to scroll
2. Text is not pretty-aligned on the banner in a mobile view: 
mobile banner
3. Tokens list menu is hidden under the footer
image

Thanks!

**Comment by fairlighteth** [comment]
Point 3 I couldn't replicate:
Screen Shot 2022-11-23 at 13 14 11

**Comment by elena-zh** [comment]
Hey @fairlighteth , regarding case 3, I reproduced in a real device iPhone 12 mini. 
You can also reproduce it in Browserstack 
image

Case 1: I have filled in the form with tokens, amounts to trade, so it became expanded. Afterwards I have been able to see this issue: 
image
What I don't like here: 
 - when the page becomes expanded, a lot of unnecessary space appears at the bottom of the page. 
 -  And a user needs to scroll down in order to see the Settlements contract link given the fact that we have some space above the WEB link
image
 - when the form is collapsed, the links are displayed somewhere in the middle of the page (not at the bottom)
image

So here I'd propose whether to keep them like on the Desktop version (at the button, in 1 line)
OR
like in the mobile view
image

However, it is my personal opinion, and can be skipped :)

**Comment by fairlighteth** [comment]
@elena-zh Thank you for the suggestions. I've been working on some optimizations and it should be much better now:

https://user-images.githubusercontent.com/31534717/203779104-d2f04461-6b4e-49a1-9fb1-956ee42e34cd.mov

**Comment by elena-zh** [comment]
Hey @fairlighteth , the footer looks awesome!
The fees banner also looks much better to me, however, I still can find nitpicks in terms of text/icons alignments (IMO, seems like all  elements aligned a bit differently). But it does not worth an effort
image

Case 1 is still not fixed. Should I create a separate issue for this? 

Thanks!

**Comment by fairlighteth** [comment]
@elena-zh Yes please, thank you!

File: src/custom/components/SwapWarnings/index.tsx (hunk 2 above)
Comment (anxolin) [reply src/custom/components/SwapWarnings/index.tsx:42]: The props names are a bit strange. They should follow the pattern `<Name>Props`

In this case, I guess should be `WarningContainerProps` 

Same for `export const HighFeeWarning = (props: WarningProps) => {`

That one should be called `HighFeeWarningProps`

**Review by elena-zh — APPROVED** [review]
Approved with #1495  issue

**Review by anxolin — APPROVED** [review]

**Pull request merged**